### PR TITLE
Updating Verify Email UI

### LINKED ISSUE #1973: Updating Verify Email UI  Similar to Eventyay
**I'm submitting a ...**
- [x] feature request

**Current behaviour:**

**Expected behaviour:**

**Steps to reproduce:**

**Screenshots**

**Would you like to work on the issue?**

Yes.

### PR DESCRIPTION
**Pull request by sk9331657**
Fixes #1973 

#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia) and my PR follows them.
- [x] My branch is up-to-date with the Upstream `development` branch.
- [x] I have added necessary documentation (if appropriate)

#### Changes proposed in this pull request:

- Updating Verify Email UI
-

### Screenshots for the change

-
screenshot from 2019-01-20 00-23-17

-

### CODE REVIEW

**Comment by sk9331657** [comment]
@abishekvashok  Review this also.

File: frontend/app/templates/components/verify-mail.hbs [hunk 1]
```diff
@@ -1,3 +1,3 @@
-<div class="top-margin">
+<div class=" ui info message top-margin">
```
Comment (abishekvashok) [inline frontend/app/templates/components/verify-mail.hbs]: Can you intend this? (Remove space before `ui`)

**Comment by sk9331657** [comment]
@abishekvashok  Done/

**Comment by sk9331657** [comment]
@fossasia/badgeyay-admins  Please reviw

**Pull request merged**